README.md

```diff
@@ -22,12 +22,13 @@ servers and syncing data across devices and applications.
 
 To develop remoteStorage-enabled apps, you need to have a
 remoteStorage-compatible storage account. We recommend
-[reStore](https://github.com/jcoglan/restore) for running a local test server.
```

Comment:
Can't we just remove all recommendations from the README, and move them into the wiki?